plugins/woocommerce/src/Gateways/PayPal/Request.php

```diff
@@ -0,0 +1,703 @@
+<?php
+/**
+ * Class Request file.
+ *
+ * @package WooCommerce\Gateways
+ */
+
+declare(strict_types=1);
+
+namespace Automattic\WooCommerce\Gateways\PayPal;
+
+use Automattic\WooCommerce\Enums\OrderStatus;
+use Automattic\Jetpack\Connection\Client as Jetpack_Connection_Client;
```

Comment:
For consistency, we might want to use `JetpackConnectionClient` as the imported class name.